apps/sms/services/js/moz_settings/moz_settings_shim.js

```diff
@@ -0,0 +1,74 @@
+/* -*- Mode: js; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- /
+/* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
+
+/* global bridge */
+
+/* exported MozSettingsShim */
+
+(function(exports) {
+'use strict';
+
+/**
+ * Name of the service for mozSettings API shim.
+ * @type {string}
+ */
+const SERVICE_NAME = 'moz-settings-shim';
+
+/**
+ * Array of method names that need to be exposed for API shim.
+ * @type {Array.<string>}
+ */
+const METHODS = ['get', 'set', 'addObserver'];
+
+var mozSettings = null;
+
+var service = null;
+
+var MozSettingsShim = {
+  init(settings) {
+    if (!settings) {
+      return;
+    }
+
+    mozSettings = settings;
```

Comment:
Yeah I considered this change for message shim before. I understand it will work since the worker only API(`importScript`) won't be triggered because libs should be loaded already for the iframe. But should we add some error handling if we do face some problem if the libs could not be loaded correctly for shim(iframe) case in `BridgeServiceMixin`?